src/parse.c

```diff
@@ -95,7 +95,7 @@ load_yaml_from_fd(int input_fd, yaml_document_t* doc, GError** error)
 
     // LCOV_EXCL_START
 file_error:
-    g_set_error(error, G_FILE_ERROR, errno, "Error when opening FD %d: %s", input_fd, g_strerror(errno));
+    g_set_error(error, NETPLAN_FILE_ERROR, errno, "Error when opening FD %d: %s", input_fd, g_strerror(errno));
```

Comment:
There might be a conflict with `errno` and our error codes defined in the `NETPLAN_FILE_ERRORS` domain. Here and below.

Should we have a `NETPLAN_FILE_ERROR.NETPLAN_ERROR_ERRNO` code, providing the errno value similarly to "filename/line/colum" of our parsing errors as part of the error message, so it can be parsed out?